### PR TITLE
Intalling the techno files for gf180mcu_c4m from CRL.

### PR DESCRIPTION
**Pull request by jpc-lip6**
Continuation of the ``wip-gf180mcu-2`` branch (skipped botched attempt with ``wip-gf180mcu_c4m-3``).

### CODE REVIEW

**Review by robtaylor — APPROVED** [review]

File: cumulus/src/designflow/technos.py [hunk 1]
```diff
@@ -186,6 +186,65 @@ def setupCMOS45 ( useNsxlib=False, checkToolkit=None, cellsTop=None ):
...
+    #ShellEnv.RDS_TECHNO_NAME = (Where.checkToolkit / 'etc' / 'FreePDK45.rds').as_posix()
+
+    path = None
+    for pathVar in [ 'PATH', 'path' ]:
+        if pathVar in os.environ:
+            path = os.environ[ pathVar ]
+            os.environ[ pathVar ] = path + ':' + (Where.allianceTop / 'bin').as_posix()
+            break
+
+
```
Comment (robtaylor) [inline cumulus/src/designflow/technos.py:247]: alliance as a dependency?

File: cumulus/src/designflow/technos.py [hunk 2]
```diff
@@ -447,3 +506,69 @@ def setupGF180MCU_GF ( checkToolkit=None, pdkTop=None ):
...
+    Yosys.setLiberty( liberty )
+    #ShellEnv.RDS_TECHNO_NAME = (Where.checkToolkit / 'etc' / 'FreePDK45.rds').as_posix()
+
+    path = None
+    for pathVar in [ 'PATH', 'path' ]:
+        if pathVar in os.environ:
+            path = os.environ[ pathVar ]
+            os.environ[ pathVar ] = path + ':' + (Where.allianceTop / 'bin').as_posix()
+            break
```
Comment (robtaylor) [inline cumulus/src/designflow/technos.py:574]: alliance a dependency?

**Review by robtaylor — CHANGES_REQUESTED** [review]
This seems to clash with #76

**Comment by jpc-lip6** [comment]
This should have be already integrated trough the merge of ``rebase-analog-fixes``

**Pull request closed without merging**